shiny/experimental/ui/_sidebar.py

```diff
@@ -144,13 +137,14 @@ def layout_sidebar(
     )
 
     res = div(
-        sidebar_dependency(),
-        sidebar_js_init(),
         {"class": "bslib-sidebar-layout"},
         {"class": "sidebar-right"} if right else None,
         {"class": "sidebar-collapsed"} if sidebar.open == "closed" else None,
         *contents,
```

Comment:
```suggestion
        main,
        sidebar.tag,
        sidebar.collapse_tag,
```

And remove `contents` definition above.